File: help/eng_gb/exercicios_da_ficha_10/Ficha10/Ficha10_001/Ficha10_001.cs
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

using System.IO;
using System;
public class Ficha10_001 {

    //Main Function
    static void Main() {
        int i = 0 ;
        while(true ){
            Console.Write("Positive number : " );
            i = int.Parse(Console.ReadLine());
            if(i > 0 ){
                break;
            }
            Console.Write("Error in reading\n\n" );
        }
        Console.Write("Entered value = " + i );
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 12 19:32

File: help/eng_gb/exercicios_da_ficha_10/Ficha10/Ficha10_001/Ficha10_001.java
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha10_001 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        int i = 0 ;
        while(true ){
            System.out.print("Positive number : " );
            i = Integer.valueOf(keyb.nextLine());
            if(i > 0 ){
                break;
            }
            System.out.print("Error in reading\n\n" );
        }
        System.out.print("Entered value = " + i );
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 12 19:32
